android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt

```diff
@@ -52,6 +52,34 @@ class BleDidcommModule(private val context: ReactApplicationContext) :
         }
     }
 
+    @ReactMethod
+    fun stopCentral(
```

Comment:
I think this should be renamed to "shutdownCentral" as not it kind of indicates to stop scanning.

Reply:
fair point.